modules/gitops-repo-server-properties.adoc

```diff
@@ -23,6 +23,9 @@ The following properties are available for configuring the Repo server component
 |`execTimeout` | `180` |Execution timeout in seconds for rendering tools (for example Helm or Kustomize).
 |`env` | `__<empty>__` |Environment to set for the repository server workloads.
 |`replicas` | `__<empty>__` |The number of replicas for the Argo CD Repo server. Must be greater than or equal to `0`.
+|`volumes` | `__<empty>__` |Configures additional volumes used for the repo server deployment. This field is optional.
+|`volumeMounts` | `__<empty>__` |Configures additional volume mounts used for the repo server deployment. This field is optional.
+|`enabled` | `true` |The flag used to enable the repo server during Argo CD installation.
```

Comment:
There is a known issue with this field
https://issues.redhat.com/browse/GITOPS-3667

Reply:
Ok, I have removed the content as per your suggestion. Fixed.